packages/neo-one-client-common/src/types.ts

```diff
@@ -1,5 +1,6 @@
 // tslint:disable deprecation no-any
 import { Monitor } from '@neo-one/monitor';
+import { Omit } from '@neo-one/utils';
```

Comment:
Ah I should've mentioned this - can you rename this to `OmitStrict`? TypeScript 3.5 will have an `Omit` type that's not quite the same.